pyproject.toml

```diff
@@ -39,6 +39,7 @@ dependencies = [
     "httpx>=0.27.2",
     "logfire-api>=1.2.0",
     "openai>=1.54.3",
+    "groq>=0.12.0",
```

Comment:
we should probably make all models (including OpenAI) optional groups.

Reply:
Probably a different PR, but agreed

Reply:
adding the group should be as simple as

```bash
uv add --optional groq groq
```